src/Render.cc

```diff
@@ -73,14 +93,14 @@ namespace drafter {
         return body;
     }
 
-    Asset renderPayloadSchema(const Payload& payload) {
-        Asset schema = payload.schema;
+    Asset renderPayloadSchema(const SectionInfo<Payload>& payload) {
+        Asset schema = payload.section.schema;
         RenderFormat renderFormat = JSONSchemaRenderFormat;
 
-        if (!payload.schema.empty() || payload.attributes.empty() || renderFormat == UndefinedRenderFormat) {
-            return schema;
+        if (!payload.section.schema.empty() || payload.section.attributes.empty() || renderFormat == UndefinedRenderFormat) {
+            return payload.section.schema;
         }
 
-        return schema;
+        return payload.section.schema;
```

Comment:
> The condition is useless...

I left it there, because we will be using it when we are rendering JSON Schema. Check how `renderPayloadBody` is.